src/index.html

```diff
@@ -11,7 +11,178 @@
+          <label for="check">I accept the term of the agreement</label>
+          <input 
+            type="checkbox" 
+            id="check" 
+            name="accept" 
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <div>
+          <label for="mail">E-mail:</label>
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            id="mail" 
+            autocomplete="off" 
+            required
+            class="inpstyles"
+          >
+        </div>
+        <div>
+          <label for="pasword">Password:</label>
+          <input 
+            type="password" 
+            name="password" 
+            minlength="5" 
+            maxlength="12" 
+            id="pasword" 
+            autocomplete="off" 
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend for="animals">An interesting fact about you!</legend>
+        <div class="inpstyles">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="yesno" 
+              value="y"
+            >
+          Yes
+          </label>
+          <label>
+            <input 
+              type="radio" 
+              name="yesno" 
+              value="n"
+            >
+          No
+          </label>
+        </div>
+     
+        <div>
+          <label for="colors">What is your favorite color?</label>
+          <input 
+            type="color" 
+            name="color" 
+            id="colors"
+            class="inpstyles"
+          >
+        </div>
+        <div>
+          <label for="times">What time do you go to bed?</label>
+          <input 
+            type="time" 
+            name="time" 
+            id="times"
+            class="inpstyles"
+            step="2"
+          >
+        </div>
+        <div class="inpstyles">
+          <label for="car">What are your favorite brand of cars?</label>
+          <select 
+            name="cars" 
+            id="car" 
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+        <div>
+          <label for="rate">How do you rate our work?</label>
+          <input 
+            type="range" 
+            name="range" 
+            minlength="0" 
+            maxlength="10" 
+            value="0" 
+            id="rate"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Additional info:</legend>
+        <div class="inpstyles">
+          <label for="comm">Comments:</label>
+          <textarea 
+            name="feedback" 
+            id="comm" 
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea 
            name="feedback" 
            id="comm" 
            autocomplete="off"
          ></textarea>
```
![image](https://user-images.githubusercontent.com/97547824/201410538-2c4300ea-2009-4272-a723-299bfafef8ed.png)